.github/workflows/release.yml

```diff
@@ -0,0 +1,225 @@
+---
+# Releases the agent
+name: release
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'The branch to release'
+        required: true
+        default: 'main'
+      version:
+        description: 'The version to release (e.g. 1.2.3). This workflow will automatically perform the required version bumps'
+        required: true
+      update_changelog:
+        description: |
+          If enabled, everything in the changelog from the "Unreleased" section will be automatically moved to a new section for the new release.
+          If disabled, the changelog needs to be prepared for the release manually before triggering this workflow.
+        type: boolean
+        required: true
+        default: true
+
+env:
+  JAVA_VERSION: 17
+  JAVA_DIST: adopt
+  MAVEN_CONFIG: >-
```

Comment:
I think https://github.com/elastic/apm-agent-java/blob/main/.mvn/maven.config duplicates this

Reply:
What does that mean?

can it be omitted?

Reply:
It can be removed, since it is already defined in the file

Reply:
https://github.com/elastic/apm-agent-java/pull/3137/commits/b4d7652bb300f06135f6887362df67161647c72c